examples/vercel-ai-chatbot/app/api/chat/route.ts

```diff
@@ -47,9 +46,19 @@ export async function POST(req: Request) {
     })
   }
 
+  const selectedWorkflowDeployment =
+    await vellum.workflowDeployments.retrieve(workflowDeploymentId)
+  const stringOutputVariables =
+    selectedWorkflowDeployment.outputVariables.filter(
+      variable => variable.type === 'STRING'
+    )
+  const targetOutputVariableName = stringOutputVariables.sort((a, b) =>
+    a.key.localeCompare(b.key)
+  )[0].key
```

Comment:
A bit of a hack so that we are always targeting a single string output variable. Trust Center bot has two string output variables and it just so happens that the one we care about is called `answer`